deployment/openshift-worker-nodes/keylime.conf

```diff
@@ -54,7 +54,7 @@ extract_payload_zip = True
 # If you set this to 'generate', keylime will create a random uuid
 # If you set this to 'hash_ek', keylime will set the UUID to the result
 # of SHA256(public EK in PEM format)
-agent_uuid = generate
+agent_uuid = worker-mrbw7
```

Comment:
I am sure you have this figured, but be mindful you don't have the same UUID on more than one node.

Reply:
Yeah, for now we have the same file for all the machines in github. When we have an ansible configuration setup it will set this per-instance.